helpers/test-data-generator.js

```diff
@@ -28,16 +28,23 @@ export async function setupAssignedTask({ colonyNetwork, colony, dueDate, domain
   const tx = await colony.makeTask(specificationHash, domain);
   // Reading the ID out of the event triggered by our transaction will allow us to make multiple tasks in parallel in the future.
   const taskId = tx.logs[0].args.id.toNumber();
+
+  await colony.setTaskRoleUser(taskId, EVALUATOR_ROLE, evaluator);
+  await colony.setTaskRoleUser(taskId, WORKER_ROLE, worker);
+
   // If the skill is not specified, default to the root global skill
   if (skill === 0) {
     const rootGlobalSkill = await colonyNetwork.getRootGlobalSkillId.call();
     if (rootGlobalSkill.toNumber() === 0) throw new Error("Meta Colony is not setup and therefore the root global skill does not exist");
-    await colony.setTaskSkill(taskId, rootGlobalSkill);
+
+    const txData = await colony.contract.setTaskSkill.getData(taskId, rootGlobalSkill);
+    const sigs = await createSignatures(colony, [MANAGER, worker], 0, txData);
+    await colony.executeTaskChange(sigs.sigV, sigs.sigR, sigs.sigS, [0, 0], 0, txData);
   } else {
-    await colony.setTaskSkill(taskId, skill);
+    const txData = await colony.contract.setTaskSkill.getData(taskId, skill);
+    const sigs = await createSignatures(colony, [MANAGER, worker], 0, txData);
+    await colony.executeTaskChange(sigs.sigV, sigs.sigR, sigs.sigS, [0, 0], 0, txData);
```

Comment:
Once you rebase on top of the work from #203 you can save some processing here by keeping the task unassigned until later and setting the task skill with just the manager signature.